modules/ROOT/pages/endpoints.adoc

```diff
@@ -1,13 +1,13 @@
 = Endpoints
 
-Endpoints define the transport protocol configurations necessary to receive or send messages with external trading partners or internal backend applications.
+Endpoints define the required transport protocol configurations to receive or send messages with external trading partners or internal backend applications.
 
 Partner Manager uses endpoints to:
 
-* Receive transmissions from partners
-* Receive application messages from the backend system
-* Specify the final destination of the message
 * Define transport protocols
+* Receive transmissions from partners
+* Receive messages from backend applications
+* Specify the final destination of messages
 * Specify where to send acknowledgments
```

Comment:
Are these in any particular order?
If not, should they be, maybe by first or most common use to last/least?
Or should they be combined by association, if one exists, like "receive transmissions from partners and messages from backend application, define transport protocols, and specify message or message acknowledgement destinations"?

Reply:
Removed bulleted list and added sentence.